build/BasePath.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Build
+{
+    internal static class BasePath
+    {
+        internal static string path { get; set; } = "..";
```

Comment:
Similar pattern followed in [Host tests](https://github.com/Azure/azure-functions-host/blob/c4a4b21b6abcf030e6ad38adb54cfc5960166ee7/test/WebJobs.Script.Tests/ScriptHostTests.cs#L376) for assigning [BaseDirectory](https://github.com/Azure/azure-functions-host/blob/c4a4b21b6abcf030e6ad38adb54cfc5960166ee7/src/WebJobs.Script/Environment/EnvironmentExtensions.cs#L22) via tests.